utils/messageqcpp/bytestream.h

```diff
@@ -76,7 +77,7 @@ class ByteStream : public Serializeable
     typedef uint16_t doublebyte;
     typedef uint32_t quadbyte;
     typedef uint64_t octbyte;
-    typedef uint128_t hexbyte;
+    typedef int128_t hexbyte;
```

Comment:
Why are we changing it to signed here?

Reply:
B/c we use uint128_t nowhere.

Reply:
For wide-DECIMAL it shouldn't matter b/c we just send 16 bytes over the wire and sign shouldn't interfere with this operation.
Moreover if we consider int128_t as a container datatype for all 16 byte DTs there is no difference if we use signed or unsigned type b/c we'll treat it as char[16].
What was your thinking when you asked?

Reply:
The code in the vicinity, e.g. octbyte, is defined as unsigned. So I thought it would make sense to defined it that way. But I agree, the signedness doesn't matter in this case.